app/javascript/react/components/BrandsIndexContainer.js

```diff
@@ -25,22 +26,22 @@ const BrandsIndexContainer = (props) => {
   const brandTiles = brands.map((brand) => {
     return(
     <div class='cell small-4 pizzaCard'>
-      <div class='card '>
+      <div class='card'>
         <div class='card-section text-center' key={brand.id}>
-          {brand.name}
+          <Link to={`/pizzas/${brand.id}`}>{brand.name}</Link>
         </div>
       </div>
     </div>
     )
   })
 
   return(
-  <div>
+  <>
     <div  className='cell small-11 text-center'>
       <h3> Brands: </h3>
     </div>
         {brandTiles}
```

Comment:
this is indented a little too much

Reply:
since {brandTiles} is not inside the div above (line 38), it should line up with it